pkg/workflow/controller/config.go

```diff
@@ -61,6 +61,8 @@ type WorkflowControllerConfig struct {
 	Secret string `json:"secret"`
 	// CycloneServerAddr is address of the Cyclone Server
 	CycloneServerAddr string `json:"cyclone_server_addr"`
+	// Notifications represents the config to send notifications after workflowruns finish.
+	Notifications []NotificationEndpoint `json:"notifications"`
```

Comment:
What kinds of notification are configured here ?

Reply:
yes